pandapower/build_branch.py

```diff
@@ -695,18 +811,28 @@ def custom_func(f, t, c):
 
 
 def _calc_r_x_from_dataframe(mode, trafo_df, vn_lv, vn_trafo_lv, sn_mva, sequence=1, characteristic=None,
-                             update_vk_values=True):
+                             update_vk_values=True, net=None):
```

Comment:
seems like on trafo_characteristic_table is needed from net. Can this be passed instead?

Reply:
*only

Reply:
linked to the above - @hilbrich to respond